test/appium/tests/atomic/chats/test_group_chat.py

```diff
@@ -0,0 +1,162 @@
+from faker import Faker
+
+from tests import marks, get_current_time
+from tests.base_test_case import MultipleDeviceTestCase
+from tests.users import chat_users
+from views.sign_in_view import SignInView
+
+
+def create_users(driver_1, driver_2, username_1=Faker().name(), username_2=Faker().name()):
+    device_1_sign_in, device_2_sign_in = SignInView(driver_1), SignInView(driver_2)
+    return device_1_sign_in.create_user(username=username_1), device_2_sign_in.create_user(username=username_2)
+
+
+def create_and_join_group_chat(device_1_home, device_2_home, errors, chat_name=Faker().name()):
```

Comment:
please use random string instead of installing an additional dependency (Faker)

Reply:
> please use random string instead of installing an additional dependency (Faker)

why do you think it's a bad idea? Faker is more flexible and friendly than usual random string